src/middleSection.js

```diff
@@ -89,8 +96,32 @@ const changeMiddleSegment = (cities) => {
     middleContainer.appendChild(card)
     document.getElementsByClassName('items-scroll')[i].style.backgroundImage = "url('/Assets/HTML&CSS/Icons_for_cities/" + value.cityName.toLowerCase() + ".svg')"
     i = i + 1
+    scrollEventPassing(card, document.querySelector('.items-container'))
   })
   updateScrollButtonsVisibility()
 }
 
-export const middleSection = { updateScrollButtonsVisibility, addCardsbySpinner, scrollItems, selectIcon, changeMiddleSegment, filterMiddleSegment }
+/**
+Method to scroll Middle section both Horizontally and Vertically
+ * @param {object} target - target element
+ * @param {object} targetContainer - underlying container
+ */
+function scrollEventPassing (target, targetContainer) {
+  target.addEventListener('wheel', (evt) => {
```

Comment:
Do we need the eventListener added in function scope? Won't it be needed to be defined in global scope?

Reply:
we are giving different value for the function parameter in different calls... i.e card and whole  scroll container as targetcontainer